pyqtgraph/dockarea/Dock.py

```diff
@@ -262,7 +262,27 @@ class DockLabel(VerticalLabel):
     sigClicked = QtCore.Signal(object, object)
     sigCloseClicked = QtCore.Signal()
 
-    def __init__(self, text, closable=False, fontSize="12px"):
+    def __init__(self, text, closable=False, fontSize=12, tabColor='#4a5c96'):
```

Comment:
If you want the default background color / radius / font size changed, that requires a bit more discussion.. I suggest keeping these as they were.